proto/daemon/v1alpha/sites.proto

```diff
@@ -0,0 +1,56 @@
+syntax = "proto3";
+
+package com.mintter.daemon.v1alpha;
+
+import "site/v1alpha/site.proto";
+
+option go_package = "mintter/backend/genproto/daemon/v1alpha;daemon";
+
+// Sites API allows to configure the remote Mintter Sites inside the local app.
+// To interact with the existing site, the local daemon will expose the Site API
+// interface as well (see /proto/site/v1alpha/site.proto), requiring a gRPC metadata
+// header with the hostname of the actual site to know the endpoint where to proxy the request.
+service Sites {
+  // Adds a site configuration to the local app.
+  rpc AddSite(AddSiteRequest) returns (SiteConfig);
+
+  // Lists configured sites.
+  rpc ListSites(ListSitesRequest) returns (ListSitesResponse);
+}
+
+// Request to add a site.
+message AddSiteRequest {
```

Comment:
Yes, the local backend will be responsible for validating the incoming request.

Reply:
I don't get it. Since the role on the site is provided by the `SiteConfig` [response](https://github.com/mintterteam/mintter/blob/build13/sites-api-codegen/proto/daemon/v1alpha/sites.proto#L55), a valid response from the server could be Ok, there is a valid site on the [hostname](https://github.com/mintterteam/mintter/blob/build13/sites-api-codegen/proto/daemon/v1alpha/sites.proto#L24) you provided but you don't have the permissions for doing anything, hence `ROLE_UNSPECIFIED`

Reply:
Well, right @horacioh I don't get what you mean either then :) I was thinking about something else, but in this AddSite request there's nothing about the role.

Anyway, ROLE_UNSPECIFIED is an invalid value, and should not be used, but should be checked for wherever the role needs to be checked (basically on the remote server only).